core/src/main/java/com/scalar/db/storage/jdbc/RdbEngineStrategy.java

```diff
@@ -325,4 +325,6 @@ default void throwIfConjunctionsColumnNotSupported(
       Set<Conjunction> conjunctions, TableMetadata metadata) {}
 
   String getTableNamesInNamespaceSql();
+
+  int getMinimumIsolationLevelForConsistentVirtualTableReads();
```

Comment:
```suggestion
  int getMinimumIsolationLevelForConsistentVirtualTableRead();
```
Should it be consistent with `isConsistentVirtualTableReadGuaranteed`?